src/settings.py

```diff
@@ -6,6 +6,22 @@
 import botconfig
 
 LANGUAGE = 'en'
+
+## TLS settings
+# If SSL_CERTFP is supplied, the bot will attempt to verify that with the server. If not, then the
+# bot will attempt certificate verification, otherwise it will abort the connection.
+# You may specify the hash algorithm to use when verifying fingerprints. If unspecified, it will
+# attempt to autodetect the hash type of each fingerprint individually. (Limited to MD5, SHA1,
+# SHA224, SHA384, and SHA512).
+# Multiple fingerprints may be comma separated for networks with multipleservers. Check your Python
+# version for suported algorithms.
+# Syntax: SSL_CERTFP = "[HASH-ALGO:]fingerprint1,fingerprint2,fingerprint3"
+SSL_VERIFY = True
+SSL_CERTFILE = None # Client cert file to connect with in PEM format; can also contain keyfile.
+SSL_KEYFILE = None # Keyfile for the certfile in PEM format. if encrypted, password will prompt on the command line.
```

Comment:
Prompting for key password on command line breaks automated restarts (e.g. via !restart and !update). Instead we should simply not support key files which require passwords to unlock.

Adding a config that specifies the unlock password is equivalent in security to not having a password, as anyone with the ability to read the key file and the bot source code will be able to unlock it either way. Filesystem permissions should be properly employed to secure the key instead.